recipes/graypy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "graypy" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dc4d46d64d12ad7061b1bac6764f69e8f03d476acb26e84b655fb116779bc499
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - graypy
+
+about:
+  home: https://github.com/severb/graypy
+  license: BSD
```

Comment:
Which BSD is it?

Reply:
Looks like it is BSD-3-Clause, see https://github.com/severb/graypy/blob/master/LICENSE

Reply:
BSD 3-clause. Will change to ```license: BSD-3-Clause```

https://github.com/severb/graypy/blob/master/LICENSE